src/app/modules/shared_models/currency_amount.nim

```diff
@@ -39,13 +39,11 @@ QtObject:
       stripTrailingZeroes: {self.stripTrailingZeroes}
       )"""
 
-  proc getAmount*(self: CurrencyAmount): float64 =
+  proc getAmount*(self: CurrencyAmount): float64 {.slot.} =
     return self.amount
 
-  proc getAmountFloat*(self: CurrencyAmount): float {.slot.}  =
-    return self.amount
-  QtProperty[float] amount:
-    read = getAmountFloat
+  QtProperty[float64] amount:
```

Comment:
I wonder what's the difference? :thinking: 

From the docu:
![image](https://github.com/status-im/status-desktop/assets/5377645/87be8b79-5c80-472b-b951-15a01c1b051d)

Reply:
Btw, there are CPUs that do _not_ support `float64` so I wonder whether we're not breaking something here. This is also the reason why Qt introduces the `qreal` (or `real` in QML) type which is an alias for either `double` or `float` based on what the CPU supports.